public/locales/de/translation.json

```diff
@@ -1,10 +1,10 @@
 {
   "intro": {
     "text": [
-      "In diesem Spiel gibt es zwei Rollen: Den Lügner und den Ermittler. Bei Beginn jeder Runde werden die Rollen getauscht.",
-      "Dem Lügner werden zwei zufällige Wikipedia Artikel präsentiert. Nur einer der beiden Artikel kann gelesen werden.",
-      "Der Ermittler sieht die Titel beider Artikel und bekommt die Möglichkeit, Fragen zu den Artikeln zu stellen.",
-      "Wird der Ermittler herausfinden, welchen Artikel sich der Lügner ausgedacht hat?"
+      "In diesem Spiel gibt es zwei Rollen: Die lügende Person und die ermittelnde Person. Bei Beginn jeder Runde werden die Rollen getauscht.",
```

Comment:
I changed the order of liar/investigator to reflect which person plays which role first. Perhaps that should taken over for the other translations, too?